vstore.class.php

```diff
@@ -2280,14 +2282,22 @@ public function productList($category=1, $np=false, $templateID = 'list', $item_
 
 		if($np === true)
 		{
+
 			$nextprev = array(
 					'tmpl'			=>'bootstrap',
 					'total'			=> $count,
 					'amount'		=> intval($this->perPage),
 					'current'		=> $this->from,
 					'url'			=> e107::url('vstore','base')."?frm=[FROM]"
 			);
-	
+
+			// Check if SEF urls are deactivated for vstore
+			$sefActive = e107::getPref('e_url_list');
+			if (empty($sefActive['vstore'])) {
+				// SEF Urls deactivated, update url
+				$nextprev['url'] = e107::url('vstore','base')."?catsef={$categoryRow['cat_sef']}&frm=[FROM]";
+			}
+
```

Comment:
See `e107_plugins/forum/e_url.php` for an example of handling legacy URLs differently to SEF.. 

eg. 

    $config['markread']  = array(
			'sef'           => '^forum/markread/{forum_id}',
			'regex'			=> 'forum/markread/([\d]*)',
			'redirect'      => '{e_PLUGIN}forum/forum.php?f=mfar&id=$1',
			'legacy'        => '{e_PLUGIN}forum/forum.php?f=mfar&id={forum_id}'
    );

Reply:
@CaMer0n I made a change, but i'm not sure if this is really the way it should be.
Using the the `$options['query']` parm for the "frm=[FROM]" doesn't work because the query parms get urlencoded which makes it fail.

Reply:
@CaMer0n I think i finally understood how to use this FROM parameter. Switched from [FROM] to --FROM-- and added it to the query option.
In my tests, it worked for both, legacy and sef urls.